### DISCUSSION TITLE
TextGeometry

### DISCUSSION THREAD

**Discussion by Clatida** [enable3d/enable3d#206] [Help / Questions]
Hi,

first of all thanks a lot for the grate work you did with Enable3D.
I would ask if TextGeometry is supported by Enable3D and, if it does, how you'd handle it.
Thanks in advance for your kind reply

Regards
Clatida

**Comment by yandeu** [comment]
Thanks 😊

You mean adding physics to a TextGeometry?

**Reply by Clatida** [reply]
<You mean adding physics to a TextGeometry?>

Yep! :) , by the way, can TextGeometry be trated as a generic shape (cube, sphere ect.) and can be compounded?
Thanks
Clatida

**Reply by yandeu** [reply]
Haven't tried it. But I think this should work 👍🏻

**Reply by Clatida** [reply]
👍 Tks
